recipes/compas_view2/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "compas_view2" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ca50d7db4b4b049fd4472125c7d23bf4d6cff09fc74d63f02748fa988f64d324
+
+build:
+  number: 0
+  skip: true  # [linux or py<36 or py>38]
```

Comment:
Why are you skipping Linux?

Reply:
got an error during the builds saying that OpenGL could not be found. we had a similar problem with the latest Mac OS but there the solution was straightforward. not sure how to fix this on linux so i figured to skip for now...

Reply:
actually the error only pops up during the test phase...

Reply:
the solution to the same problem on Mac was this https://github.com/compas-dev/compas_view2/blob/main/src/compas_view2/__init__.py#L20